UC-Berkeley-Extension/global
# Accordion component

## Asana Ticket(s)

## Description
- Built Accordion styleguide component
- Adjusted js organization and compiling
- Built Accordion Item and Accordion paragraph components

## Testing instructions
- Compile the styleguide: move to bglobal theme root, run `npm start`
- Back in project root, `ddev start` then ssh into the container
- `drush cim`, `drush cr` (or use the `refresh.sh` script in the README)
- Create a Landing Page, on the main Berkeley select option
- Add a Column Section, then an Accordion.
- Verify that it is working properly.
